test/pageobjects/UserConsent.js

```diff
@@ -0,0 +1,40 @@
+import { verifyElementCopy } from '../utils/mochaw'
+import BasePage from './BasePage.js'
+
+class UserConsent extends BasePage {
+  async consentFrameTitle() {
+    return this.$('.onfido-sdk-ui-UserConsent-consentFrame > h1')
+  }
+
+  async declineBtn() {
+    return this.$('.onfido-sdk-ui-UserConsent-secondary')
+  }
+  async acceptBtn() {
+    return this.$('.onfido-sdk-ui-Button-button-primary')
+  }
```

Comment:
As above. 

For our current Primary, Secondary buttons there is a `uiTestDataAttribute` property that you can set a name to & it will render the button element with `data-onfido-qa="elementNameForUITests"`.